src/style.css

```diff
@@ -1,3 +1,84 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  margin: 10px 50px;
+}
+
+.nav {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: 50px;
+}
+
+.nav_list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+}
+
+.nav_list li {
+  display: inline;
+  margin: 0 10px;
```

Comment:
It's better to add margin-right: 20px to each `<li>` tag and remove margin-right for the last child